portal/sign/assets/signit.php

```diff
@@ -0,0 +1,47 @@
+<?php
+// check if authenticated
+require_once(dirname(__FILE__) . "/../../../interface/globals.php");
+$thisuser = $_SESSION['authUser'];
+$thisDevice = "User:" . $thisuser;
+// don't want seesion stepped on somehow
+unset($_SESSION);
+
+// most know what this is..
```

Comment:
Is it something you eat?

Reply:
(couldn't help myself)

Reply:
It's nice to see you crack a joke. Only thing is I can't think of a witty come back.:) How about, only if you have the stomach for it!:)